modules/bootstrap-ks/Makefile

```diff
@@ -0,0 +1,353 @@
+#-- bootstrap-ks Params---#
+BOOTSTRAP_KS_REPO ?= https://${GITHUB_TOKEN}@github.com/open-cluster-management/bootstrap-ks.git
+BOOTSTRAP_KS_BRANCH ?= master
+BOOTSTRAP_KS_DEPLOY_DIR ?= bootstrap-ks
+
+#---Login/Host-cluster related env vars---#
+# API URL of the cluster running the target clusterpool, used in oc login, so port :6443 is required
+BOOTSTRAP_KS_HOST_API ?=
+# Access credentials used to log in to cluster indciated in BOOTSTRAP_KS_HOST_API
+BOOTSTRAP_KS_HOST_USER ?=
+BOOTSTRAP_KS_HOST_PASS ?=
+BOOTSTRAP_KS_HOST_TOKEN ?=
+# Namespace where Hive ClusterPool resources reside
+BOOTSTRAP_KS_HOST_NAMESPACE ?= clusterpool
+
+OC_CLUSTER_USER = BOOTSTRAP_KS_HOST_USER
+OC_CLUSTER_PASS = BOOTSTRAP_KS_HOST_PASS
+OC_CLUSTER_API = BOOTSTRAP_KS_HOST_API
+
+CLUSTER_NAME_PREFIX ?= canary
+
+# These will only have a value if the cluster has been deployed.  (Only one per type is allowed)
+BOOTSTRAP_KS_AKS_CLUSTER_NAME = $(shell cd ${BOOTSTRAP_KS_DEPLOY_DIR}/aks; cat *.json | jq -r .CLUSTER_NAME)
```

Comment:
These bare `jq` references should be `$(JQ)` - also, rather than changing working directory over to /aks, couldn't you just:
`cat ${BOOTSTRAP_KS_DEPLOY_DIR}/aks/*json`?  (Is there only ever one json file there?)

Reply:
I will fix jq calls.  And will look at changing the `cd` calls.  It is a leftover from original prototype